pkg/apis/search/cache_types.go

```diff
@@ -1,7 +1,7 @@
 package search
 
 import (
-	corev1 "k8s.io/api/core/v1"
+	policyv1alpha1 "github.com/karmada-io/karmada/pkg/apis/policy/v1alpha1"
```

Comment:
@RainbowMango What is the problem
`File is not `goimports`-ed with -local github.com/karmada-io/karmada (goimports)`